src/django/contricleaner/lib/serializers/row_serializers/row_country_serializer.py

```diff
@@ -4,8 +4,10 @@
 
 class RowCountrySerializer(RowSerializer):
     def validate(self, row: dict, current: dict) -> dict:
+        country = row.get("country", '')
```

Comment:
it's better to move this line under the `try:` block

Reply:
Done